src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -132,11 +132,17 @@ function getListBlockType(
 function getBlockMapSupportedTags(
   blockRenderMap: DraftBlockRenderMap
 ): Array<string> {
+  // Some blocks must be treated as unstyled when not present on the blockRenderMap
   const unstyledElement = blockRenderMap.get('unstyled').element;
-  return blockRenderMap
+  const defaultUnstyledSet = new Immutable.Set(['p']);
```

Comment:
We need this since even though we don't support paragrah directly internally we still want to treated pasted paragraph content as different unstyled blocks